### ISSUE TITLE
New lines when printing systems

### ISSUE THREAD

**Issue by guillaumebrunerie** [mortberg/cubicaltt#101]
When there is an error message showing a system (e.g. incompatible system, and maybe others), it would be nice if the system is printed with one component per line:
```
[ (i = 0) -> …
, (i = 1) -> …
, (j = 0) -> …
, (j = 1) -> …]
```
That would make it much easier to grasp what is going on.

**Comment by mortberg** [comment]
A quick and dirty way would be to add a "\n" in the string that we intercalate with in https://github.com/mortberg/cubicaltt/blob/master/Connections.hs#L384

Is this what you want? It won't be very pretty, but it might be better than what we have now...

**Comment by guillaumebrunerie** [comment]
Thanks, that’s a bit better although when one of the faces contains an hcomp (and a system), then both the inner and the outer systems appear at the same level of indentation which is a bit confusing.

**Comment by mortberg** [comment]
Yeah, I guess it would be nice to get the pretty printer to properly indent stuff... It would also be nice to post-process the output to use more readable names instead of `i1423827`
